question-service/route.rest

```diff
@@ -0,0 +1,26 @@
+GET http://localhost:5000/api/v1/questions
```

Comment:
is this the file for inserting data into the database? if so, it will be much more convenient if you could use a .env file so its easier for different machines to host the databases